src/config.tsx

```diff
@@ -10,3 +10,5 @@ export const ApiBaseUrls = {
   [EnvironmentsEnum.devnet]: 'https://devnet-tools.multiversx.com',
   [EnvironmentsEnum.testnet]: 'https://testnet-tools.multiversx.com'
 };
+
+export const miscApi = 'https://misc-api.multiversx.com';
```

Comment:
MISC_API ?

Reply:
Used the same naming pattern as the other constants in the file